independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/multipart/PartItem.java

```diff
@@ -24,4 +26,8 @@ public MediaType getType() {
     public Object getValue() {
         return value;
     }
+
+    public String getRawGenericType() {
```

Comment:
I don't think this name properly conveys what this does.
At the very least, we should probably document what it returns for both generic and non generic cases.

Reply:
Renamed to "firstParamType" and added comments in the method.

Reply:
Let me know if the name looks better for you